CHANGELOG.md

```diff
@@ -17,6 +17,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   which adds some fixes around cert injection failure (see also changes in 
   [0.18.1](https://github.com/cyberark/conjur-authn-k8s-client/blob/master/CHANGELOG.md#0181---2020-09-13)). [cyberark/secrets-provider-for-k8s#247](https://github.com/cyberark/secrets-provider-for-k8s/pull/247)
 
+### Fixed
+- The version that is printed at the product's startup now includes the git commit
```

Comment:
do you still want the git commit's hash after the convo with @izgeri  in slack?

> we want the version for tags _just_ to be the tag

Reply:
yes, but that effort will be bigger. i created an issue for that: https://github.com/cyberark/secrets-provider-for-k8s/issues/258

I think that we should first make this quick fix so that we don't have the hard-coded `dev` tag. the next fix will take more time and we can address that when we have the time.